ray-on-gke/user/main.tf

```diff
@@ -18,16 +18,23 @@ provider "kubernetes" {
   config_path = pathexpand("~/.kube/config")
 }
 
-provider "kubectl" {
-  config_path = pathexpand("~/.kube/config")
-}
-
 provider "helm" {
   kubernetes {
     config_path = pathexpand("~/.kube/config")
   }
 }
 
+module "kubernetes-nvidia" {
+  source           = "../../gke-platform/modules/kubernetes"
+  enable_autopilot = var.enable_autopilot
+  enable_tpu       = var.enable_tpu
+}
+
+module "kuberay-operator" {
+  source           = "../../gke-platform/modules/kuberay"
+  enable_autopilot = var.enable_autopilot
+}
```

Comment:
These are pieces that we expect to be installed by the cluster admin, but my understanding of the division was that the user module was intended to be installed by non-admin users?

I think what we may need to do is split gke-platform into 2 pieces: one to create the cluster & node pools, etc; and a second phase to set up the cluster-wide kubernetes workloads (kuberay operator & nvidia drivers)

Reply:
yes right, in that case we should have one separate section, or other way can be to convert manifests into simple helm charts and use `helm_release` instead..

if we want to go through user personas way, then we first need to define what is expected out of these users and accordingly may be create two sub folders inside workloads..  but we can brainstorm more on this on call. Feel free to schedule one.